docs/recipes/low_freq_proc.qmd

```diff
@@ -0,0 +1,183 @@
+---
+title: "Low-Frequency Processing"
+execute:
+  eval: false
+---
+
+This recipe demonstrates how DASCore can be used to apply low-frequency (LF) processing to a spool of DAS data.
```

Comment:
Maybe we can state this is equivalent to downsampling an entire spool in case someone isn't familiar with what "low-frequency processing" means.